packages/schedule/demo/stories/ScheduleStory.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import {
+  IScheduleContainerProps,
+  IUseScheduleProps,
+  IUseScheduleReturnValue,
+  ScheduleContainer,
+  useSchedule
+} from '@zendeskgarden/container-schedule';
+
+interface IComponentProps extends IUseScheduleReturnValue {
+  duration?: IUseScheduleProps['duration'];
+  loop?: IUseScheduleProps['loop'];
+}
+
+const Component = ({ delayMS, delayComplete, elapsed, loop, duration = 1250 }: IComponentProps) => (
+  <div>
+    <label>
+      <span>
+        Schedule {delayMS / 1000}s delay followed by {duration / 1000}s{' '}
+        {loop ? 'looped' : 'elapsed'} progress
+      </span>
+      <progress className="block w-full" value={delayComplete ? elapsed : undefined} />
+    </label>
+  </div>
+);
+
+const Container = ({ loop, ...props }: IScheduleContainerProps) => (
+  <ScheduleContainer loop={loop} {...props}>
+    {containerProps => <Component {...containerProps} loop={loop} />}
```

Comment:
Wouldn't `containerProps` also pass thru `loop` from the container with no need to manually pass `loop` to `Component`?

Reply:
Unfortunately, the `loop` prop is not reconstituted in the `containerProps`.  (In going through this exercise, I'm noting API discontinuity between the packages – this is one area where we could improve consistency.) So, I'm feeding it through. But worth noting that a) it is simply for string display, and b) these particular (overly) abstracted examples are a bit contrived since IRL one would either use the container or the hook and all the props would likely be available in one place.